tools/arm_pack_manager/index.json

```diff
@@ -462801,6 +462801,20 @@
                 "default": true, 
                 "start": "0x10002000",
                 "size": "0x000FE000"
+            },
+            "IRAM1": {
+                "access": {
+                    "execute": true,
+                    "non_secure": true,
```

Comment:
This permission bit is reserved for v8m platforms when I last checked, and will have no affect for v7m or v6m platforms.